src/hash-table.js

```diff
@@ -7,6 +7,45 @@ class HashTable {
     this.storage = new LimitedArray(this.limit);
     // Do not modify anything inside of the constructor
   }
+  insert(key, value) {
+    const index = getIndexBelowMax(key, this.limit);
+    if (this.storage[index] === undefined) {
```

Comment:
Try getting in the habit of using `!` when you want to check if something is undefined or null. It is more concise while also being more flexible.